Sources/NIOWebSocketServer/main.swift

```diff
@@ -122,6 +122,10 @@ private final class WebSocketTimeHandler: ChannelInboundHandler {
             self.pong(ctx: ctx, frame: frame)
         case .unknownControl, .unknownNonControl:
             self.closeOnError(ctx: ctx)
+        case .text:
+            var data = frame.unmaskedData
+            let text = data.readString(length: data.readableBytes) ?? ""
+            self.printToConsole(text: text)
```

Comment:
I don’t think we need this abstraction: we can just call `print`.

Reply:
ok, I deleted printToConsole method